tad/api/routes/tasks.py

```diff
@@ -21,21 +21,15 @@ async def move_task(
     :param moved_task: the move task object
     :return: a HTMLResponse object, in this case the html code of the card that was moved
     """
-    try:
-        # because htmx form always sends a value and siblings are optional, we use -1 for None and convert it here
-        if moved_task.next_sibling_id == -1:
-            moved_task.next_sibling_id = None
-        if moved_task.previous_sibling_id == -1:
-            moved_task.previous_sibling_id = None
-        task = tasks_service.move_task(
-            moved_task.id,
-            moved_task.status_id,
-            moved_task.previous_sibling_id,
-            moved_task.next_sibling_id,
-        )
-        # todo(Robbert) add error handling for input error or task error handling
-        return templates.TemplateResponse(request=request, name="task.html.jinja", context={"task": task})
-    except Exception:
-        return templates.TemplateResponse(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, request=request, name="error.html.jinja"
-        )
+    # because htmx form always sends a value and siblings are optional, we use -1 for None and convert it here
```

Comment:
I assume removing the try/catch is possible because this exception is now handled by the 'get_undefined_behaviour' method in the LocaleJinja2Templates?

Reply:
No, i added better exception handling reponse